### PR TITLE
Add logging to cron to see why nothing is triggered

### PR DESCRIPTION
**Pull request by krzyzacy**
ref https://github.com/kubernetes/test-infra/issues/5468

/assign @BenTheElder @cjwagner

### CODE REVIEW

File: prow/cron/cron.go [hunk 1]
```diff
@@ -147,8 +148,9 @@ func (c *Cron) addJob(name, cron string) error {
 	id, err := c.cronAgent.AddFunc(cron, func() {
 		c.lock.Lock()
 		defer c.lock.Unlock()
-		// We want to ignore second trigger if first trigger is not consumed yet.
+
 		c.jobs[name].triggered = true
+		logrus.Infof("triggering cron job %s", name)
```
Comment (cjwagner) [inline prow/cron/cron.go]: nit: Capitalize "triggering" and add periods to the log messages.

File: prow/cmd/horologium/main.go [hunk 2]
```diff
@@ -46,7 +46,8 @@ func main() {
 	}
 
 	// start a cron
-	cr := cron.New()
+	cronLogger := logrus.WithField("agent", "cron")
```
Comment (cjwagner) [inline prow/cmd/horologium/main.go]: nit: This field should be added in the constructor for `Cron`.

Comment (krzyzacy) [reply prow/cmd/horologium/main.go]: awww right :-( I was referencing somewhere that passes the entry along

Comment (cjwagner) [reply prow/cmd/horologium/main.go]: You can still make it accept a `logrus.Entry`, but the "agent" field should probably be added in the constructor.

Comment (kargakis) [reply prow/cmd/horologium/main.go]: "agent" conflicts with pj.Spec.Agent. Should this be just `client`?

Comment (cjwagner) [reply prow/cmd/horologium/main.go]: That makes sense to me. If "agent" is used to log `pj.Spec.Agent` then this should be changed: https://github.com/kubernetes/test-infra/pull/5442/files#diff-0e5c8bf65ff28507861c63b6bbe7274aR88

Comment (krzyzacy) [reply prow/cmd/horologium/main.go]: done

Comment (kargakis) [reply prow/cmd/horologium/main.go]: We still don't log that field but it's possible to do it in the future.

**Pull request merged**